importer/DkBygningDa.py

```diff
@@ -14,15 +14,25 @@ def set_location(self):
         """
         Use self.municipality because IT'S PLACE NOT MUNICIPALITY
         (that's adm2)
+        Look it up in list of Danish placenames.
+        Some are wikilinked, but we can't rely on it
+        because of items like [[København K]], which redirect
+        to an article about postal codes...
         """
         place_item = False
+        placename_dict = self.data_files["settlements"]
         if self.has_non_empty_attribute("by"):
             place = self.by
             if utils.count_wikilinks(place) == 1:
```

Comment:
If you are just getting rid of the links I assume you also do not need to count them?

Please also see the comment directly on the PR about alternatives to completely getting rid of the wikilink.